### PR TITLE
Upstream bot

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

TG added a Downstream bot, this just spins that around.

## Why It's Good For The Game

No, I don't wanna go upstream!!

## Changelog
:cl:
add: An Upstream tourist bot, now at your local Bar or Kitchen!
/:cl:

### CODE REVIEW

**Comment by Helianthus00** [comment]
This might just be funny enough to warrant the tg edit

**Comment by optimumtact** [comment]
IM going to kill you John

**Comment by Qustinnus** [comment]
owned?

File: fulp_modules/malfunction_bot/upstream_bot.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			/obj/item/toy/crayon/purple = 1,
+			/obj/item/food/canned/peaches/maint = 6,
+			/obj/item/food/tofu = 1,
+		),
+		/datum/venue/bar = list(
+			/datum/reagent/consumable/ethanol/beer = 1,
+			/datum/reagent/consumable/failed_reaction = 1,
+			/datum/reagent/spraytan = 1,
+			/datum/reagent/reaction_agent/basic_buffer = 1,
+			/datum/reagent/reaction_agent/acidic_buffer = 1,
```
Comment (Iamgoofball) [inline fulp_modules/malfunction_bot/upstream_bot.dm:29]: can the bar even make any of these last three?

Comment (JohnFulpWillard) [reply fulp_modules/malfunction_bot/upstream_bot.dm:29]: I didn’t even change any of the orders, it’s the same as the downstream one

**Pull request merged**